user-service/src/main/java/com/adavydenko/userservice/UserServiceApplication.java

```diff
@@ -1,22 +1,23 @@
-package com.adavydenko.tictactoe;
```

Comment:
i would keep tictactoe as part of the package.. you may also make it as part of your artifact group in common build.gradle file.